### PR TITLE
Allow users to attach files to message

### PR DESCRIPTION
**Pull request by pardhaponugoti**
Summary: integrates with IPFS/Filecoin to allow users to optionally add files to the message

### CODE REVIEW

File: contracts/AckToEarn.sol [hunk 1]
```diff
@@ -24,6 +24,7 @@ contract AckToEarn is Ownable, ReentrancyGuard {
         address recipient;
         string responseEmailAddress;
         string message;
+        string fileCid;
```
Comment (FaybianB) [inline contracts/AckToEarn.sol:27]: Should this be an array of strings, in case multiple files are uploaded?

Comment (pardhaponugoti) [reply contracts/AckToEarn.sol:27]: hm let me check

Comment (pardhaponugoti) [reply contracts/AckToEarn.sol:27]: i believe filecoin just makes one folder and gives you a `fileCid` with that folder's id but worth double checking

Comment (pardhaponugoti) [reply contracts/AckToEarn.sol:27]: i can only choose one file on my machine

**Pull request merged**